pkg/server/server.go

```diff
@@ -186,6 +183,9 @@ func (s *Server) Run(ctx context.Context) error {
 
 		if s.Options.Extra.ShardName == tenancyv1alpha1.RootShard {
 			logger.Info("bootstrapping root workspace phase 0")
+
+			s.RootShardKcpClusterClient = s.KcpClusterClient
```

Comment:
seems this is not needed as we assign in the config.go

Reply:
fixed